cpp/keybinds/key.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+#include <functional>
+
+#include "key.h"
+
+namespace openage {
+namespace keybinds {
+
+key_t::key_t(SDL_Keycode key)
+    :
+    key(key),
+    mod(KMOD_NONE)
+{
```

Comment:
tab indent again. For the initializer list init and more hints about the codestyle we're using (because of consistency and beautyfication reasons) please have a look at this: https://github.com/SFTtech/openage/blob/master/doc/code_style/mom.cpp#L131

all your new files are space indented :wink: